storagemanager/users/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.db.models.signals import post_save
+from django.contrib.auth.models import User
+from django.dispatch import receiver
+from .models import Profile
+
+
+@receiver(post_save, sender=User)
+def create_profile(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user=instance)
+
+
+@receiver(post_save, sender=User)
+def save_profile(sender, instance, **kwargs):
```

Comment:
:thinking: Do we need to save `Profile` each time we save `User`? We can't save profile model instance directly, right? What happens if you don't have this signal?